index.js

```diff
@@ -1,20 +1,440 @@
+
+/*------------------------------------Create ITEMs (passing all properties down from ITEM CLASS ) ---------------------------------------------------*/
+// name of item,
+// description,
+// action (verbs allowed to be used on the item)
+// actionOutcome (response prompt after action is complete),
+// takeable (can the item be taken into player inventory)
+
+/*------------------------------------Create Player Object ---------------------------------------------------*/
+// The object -- playerObject -- has two KEY-VALUE pairs: currentRoom is Key and "forest" is value. playerInventory is key and empty array is value.
+let playerObject = {
+  currentRoom: "forest",
+  playerInventory: [],
+  playerInstructions: `\nPlayer Instructions:\nAs you move along the path, you will have opportunities to engage with different items in different locations.\n
+  To declare your next move, type an action (verb) followed by the item (noun).\n
+  Index of game verbs:
+  "move" - allows you to move from one location to the next
+  "describe" - allows you to receive a description of the room you are in
+  "use" - allows you to utilize an item
+  "examine" - allows you to examine an item
+  "take" - allows you to take an item. Once an item is taken it is stored in your inventory\n
+  For example (and don’t worry, this is not a spoiler), If you were to come across a branch on the path,\n you have the following options: “take branch”, “use branch” or "examine branch".\n
+  
+  A few other notes to keep in mind while you’re on the path:
+  "leave" enables you to leave the program;
+  "i" allows you to check your inventory at any point during the game;
+  "help" will return you to this instructional message.
+  
+  Ok, that seems to be all you need to know...for now…\n`
+};
+
+let door = new Item(
+  "door",
+  "a moss-encased door frame with a tiny window, also encased in moss",
+  ["use", "examine"], //allowable actions on this item include both "use" and "examine",
+  "Sorry, this door doesn’t open like other doors--just with a turn of a handle or a press of a button. The moss will help you if you ask"
+  // not takeable so will just default to false
+);
+
+let moss = new Item(
+  "moss",
+  "green, velvety, and vast with an affinity for riddles.",
+  ["use", "examine"], //allowable actions on this item include both "use" and "examine"
+  "The moss whisper, “solve this riddle and you may enter. “ What is always in front of you but can’t be seen?"
+  // not takeable so will just default to false
+);
+
+let teacup = new Item(
+  "teacup",
+  "White china with a floral pattern. The handle is slightly chipped. The remnants of a lipstick kiss rests on the rim.",
+  "examine",
+  "Silly, there is no tea here to steep, nor water to boil, nor means to boil it. Go find something more useful..."
+  // Not Takeable -- How do I include message for this particular untakeable item? E.g. "So sorry, but this teacup belongs to this room. You’ll have to find your own."
```

Comment:
You might consider having handling for two different kinds of error. One checks if the target is not an item, and the other checks the location of the item. They could return different messages that indicate this. "That doesn't exist" vs "that's not here."